.build/BuildHelper/Build-FinOpsModule.ps1

```diff
@@ -0,0 +1,82 @@
+function Build-FinOpsModule
+{
+    [CmdletBinding()]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [string]
+        [ValidateScript({$_ -match '^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$'})]
+        $Version,
+
+        [Parameter()]
+        [ValidateSet('alpha', 'preview')]
+        [string]
+        $PrereleaseTag
+    )
+
+    $rootPath = (Get-Item -Path $PSScriptRoot).Parent.FullName
+    $moduleName = 'FinOpsToolkit'
+    $moduleFullName = "$moduleName.psm1"
+    $modulePath = Join-Path -Path $rootPath -ChildPath "src/powershell/$moduleFullName"
+    $stringsPath = Join-Path -Path $rootPath -ChildPath 'src/powershell/en-US'
+    $releasePath = Join-Path -Path $rootPath -ChildPath "release/FinOpsToolkit/$Version"
+    $scriptPath = Join-Path -Path $rootPath -ChildPath 'src/scripts'
+    $manifestPath = Join-Path -Path $releasePath -ChildPath 'FinOpsToolkit.psd1'
+
+    # Make sure we can import module properly. Capture exported functions.
+    try
+    {
+        Import-Module -FullyQualifiedName $modulePath -ErrorAction 'Stop'
+        $exportedCommands = Get-Command -Module $moduleName
+        Remove-Module -Name $moduleName
+    }
+    catch
+    {
+        throw ("Error importing module at path: '{0}', {1}" -f $modulePath, $_.Exception.Message)
+    }
+
+    # Create release directory
+    Push-Location -Path $scriptPath
+    ./New-Directory $releasePath
+    Pop-Location
+
+    $manifestProperties = @{
+        ModuleVersion     = $Version
+        Path              = $manifestPath
+        Guid              = '00f120b5-2007-6120-0000-b03e1254e770'
+        Author            = 'Microsoft Corporation'
+        CompanyName       = 'Microsoft Corporation'
+        Description       = 'FinOps toolkit - Cmdlets to deploy and manage resusable FinOps solutions in Azure. This module is compatible with PowerShell and Windows PowerShell. For more information about the FinOps toolkit, please visit https://aka.ms/finops/toolkit'
```

Comment:
nit: "Commands" instead of "cmdlets"?
```suggestion
        Description       = 'FinOps toolkit - Commands to deploy and manage resusable FinOps solutions in the Microsoft Cloud. This module is compatible with PowerShell and Windows PowerShell. For more information about the FinOps toolkit, please visit https://aka.ms/finops/toolkit'
```